tests/test_metadata.py

```diff
@@ -119,11 +119,11 @@ def test_get_metadata_from_distribution():
         "repository",
         "version",
     }
-    # ignore version and description
+    # ignore version because not fixed
```

Comment:
what do you mean `not fixed`? fixed as in repaired, or fixed as in unchanging?

Reply:
not unchanging, I'll rephrase it